workshop_2018_2019/angular_workshop.md

```diff
@@ -2,22 +2,22 @@
 Angular is front-end frame webframework mostly used to build single page applications. For this workshop, we will be using Angular to show your brain-art in the browser.
 
 ## Install Node.js, npm, Angular
-You must have Node.js and npm package manager installed in your machine before getting started with angular. After you have those two, install angular.
+You must have Node.js and npm package manager installed in your machine before getting started with angular. After you have those two, install angular. Node.js and npm can be installed from here: https://nodejs.org/en/download/. To install angular: run this command in your terminal (no quotes): "npm install -g angular-cli"
```

Comment:
- Step-by-step instructions are easier to follow than a para of text
- You can format the terminal command as a terminal command
- Also, Angular, not angular